src/corebluetooth.cs

```diff
@@ -7,6 +7,8 @@
 //
 // Copyright 2011-2013 Xamarin Inc
 //
+using System.ComponentModel;
+
```

Comment:
extra line intended?

Reply:
yes, I like to separate the bcl using from ours